src/constants/logURLTemplates.ts

```diff
@@ -53,7 +53,7 @@ const getResmokeLogURL = (
 };
 
 enum originToType {
```

Comment:
[nit] TypeScript Enums are usually capitalized (e.g. `OriginToType.Agent`), but no need to block merging on this.